config/config_embed.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	_ "embed"
+)
+
+// MainnetConfig is the Neo N3 mainnet configuration.
+//
+//go:embed protocol.mainnet.yml
+var MainnetConfig []byte
```

Comment:
It's `config.MainnetConfig` which is a bit duplicating. I'd suggest either make it `config.Mainnet` or `embedded.MainnetConfig` (since we already have a `pkg`-level `config` package). The first option seems to be slightly better for me.